lighthouse-core/computed/metrics/lantern-largest-contentful-paint.js

```diff
@@ -25,6 +25,18 @@ class LanternLargestContentfulPaint extends LanternMetric {
     };
   }
 
+  /**
+   * @param {Node} node
+   * @return {boolean}
+   */
+  static isLowPriorityImageNode(node) {
+    if (node.type !== 'network') return false;
+
+    const isImage = node.record.resourceType === 'Image';
+    const isLowPriority = node.record.priority === 'Low' || node.record.priority === 'VeryLow';
+    return isImage && isLowPriority;
+  }
+
   /**
    * TODO: Validate.
```

Comment:
remove this TODO now?

Reply:
heh, good call. we're suddenly in a situation where LCP is arguably the *most* accurate 😆